### ISSUE TITLE
Setting dayMinWidth causes error "NowTimer(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null."

### ISSUE THREAD

**Issue by harryy2510** [fullcalendar/fullcalendar-react#80]
### Bug Reports
I am using resource time grid day plugin with scroll grid. when setting dayMinWidth, it throws error

```
  const calendarProps = {
    plugins: [ scrollGridPlugin, resourceTimeGridPlugin ],
    initialView: 'resourceTimeGridDay',
    resources: [
      { id: 'a', title: 'Room A' },
      { id: 'b', title: 'Room B'},
      { id: 'c', title: 'Room C' },
      { id: 'd', title: 'Room D' },
      { id: 'e', title: 'Room E' },
      { id: 'f', title: 'Room F'},
      { id: 'g', title: 'Room G' },
      { id: 'h', title: 'Room H' },
      { id: 'i', title: 'Room I' },
      { id: 'j', title: 'Room J'},
      { id: 'k', title: 'Room K' },
      { id: 'l', title: 'Room L' },
      { id: 'm', title: 'Room M' },
      { id: 'n', title: 'Room N'},
      { id: 'o', title: 'Room O' },
      { id: 'p', title: 'Room P' },
    ],
    dayMinWidth: 240
  }
```

https://stackblitz.com/edit/react-ts-nngqvn?file=App.tsx

**Comment by hitmanche** [comment]
Yes i get this error with react. I have tried many times on react by looking at the demo already, will there be an update about this, there are some errors like this, if these errors will be fixed, I will buy the premium version but I am afraid to buy it right now because it is missing.

**Comment by tmhao2005** [comment]
the issue seems to come from following where the `<NowTimer />` component doesn't return anything at the end (most likely `null` in this case)

https://github.com/fullcalendar/fullcalendar/blob/e028f8b414059393353da30245103f117404242a/packages/timegrid/src/TimeColsView.tsx#L233

**Comment by SrAnthony** [comment]
I fixed by adding `return null` inside node_modules/@fullcalendar/timegrid/main.js, line 448:
![Screen Shot 2020-07-22 at 09 30 28](https://user-images.githubusercontent.com/14919996/88176679-0426ba00-cbfe-11ea-9bd3-cc4e922f2c36.png)

I also used patch-package.

**Comment by arshaw** [comment]
this is fixed in v5.2.0 !
